localization/camera/src/camera_params.cc

```diff
@@ -266,6 +268,27 @@ void camera::CameraParameters::DistortCentered(Eigen::Vector2d const& undistorte
     }
     *distorted_c = (optical_offset_ - distorted_half_size_) +
       conv * norm.cwiseProduct(focal_length_);
+  } else if (distortion_coeffs_.size() == 2) {
+    // colmap radial fisheye model
+    double k1 = distortion_coeffs_[0];
+    double k2 = distortion_coeffs_[1];
+
+    // To relative coordinates
+    Eigen::Vector2d norm = undistorted_c.cwiseQuotient(focal_length_);
+    double r = norm.norm();
+    if (r > std::numeric_limits<double>::epsilon()) {
+      double theta = atan(r);
+      double theta2 = theta * theta;
+      double thetad = theta * (1 + k1 * theta2 + k2 * theta2 * theta);
```

Comment:
I think this should read ".....k2*theta2*theta2"

Reply:
Good catch, thanks! Fixed